pages/lists/[listId]/activity.tsx

```diff
@@ -25,11 +22,6 @@ interface ContributorListPageProps {
   list?: DBList;
   numberOfContributors: number;
   isError: boolean;
-  activityData: {
```

Comment:
All initial graph data from the server-side has been removed so only essential list information is loaded from the server-side.